### ISSUE TITLE
Unable to Download CNI plugin for 1.16

### ISSUE THREAD

**Issue by pratheep-kumar** [awslabs/amazon-eks-ami#497]
**What happened**:
While creating AMI, We are facing issue in running 
https://github.com/awslabs/amazon-eks-ami/blame/822aa4ab3894706f2b309b88df0d8c886a6e782d/scripts/install-worker.sh#L182

**What you expected to happen**:
It should be able to wget that plugin

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

URL formed: https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-amd64-v0.8.6.tgz
Correct URL: https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz 

**Environment**:
- AWS Region:
- Instance Type(s):
- EKS Platform version (use `aws eks describe-cluster --name <name> --query cluster.platformVersion`):
- Kubernetes version (use `aws eks describe-cluster --name <name> --query cluster.version`):
- AMI Version:
- Kernel (e.g. `uname -a`):
- Release information (run `cat /etc/eks/release` on a node):

```
```

**Comment by maroda** [comment]
It appears the CNI release file naming changed in v0.8:

https://github.com/containernetworking/plugins/releases/tag/v0.8.2

It isn't limited to amazon-eks-ami v1.16, it also throws the same error in other build versions (here with `make 1.15`):

```
    amazon-ebs: --2020-06-30 19:52:02--  https://github.com/containernetworking/plugins/releases/download/v0.8.6/cni-plugins-amd64-v0.8.6.tgz
    amazon-ebs: Resolving github.com (github.com)... 192.30.255.112
    amazon-ebs: Connecting to github.com (github.com)|192.30.255.112|:443... connected.
    amazon-ebs: HTTP request sent, awaiting response... 404 Not Found
    amazon-ebs: 2020-06-30 19:52:02 ERROR 404: Not Found.
```

The string *-linux* after _cni-plugins_ is new, and missing.

**Comment by mogren** [comment]
Resolved in #500 (and follow up #520)
